app/src/main/java/org/fossasia/openevent/general/social/SocialLinksViewModel.kt

```diff
@@ -19,6 +19,7 @@ class SocialLinksViewModel(private val socialLinksService: SocialLinksService) :
     val socialLinks: LiveData<List<SocialLink>> = mutableSocialLinks
     private val mutableError = SingleLiveEvent<String>()
     val error: LiveData<String> = mutableError
+    val mutableInternetError: MutableLiveData<Boolean> = MutableLiveData<Boolean>()
```

Comment:
This isn't used anywhere

Reply:
I have used it in SocialLinksFragment.kt: 105 & 107

Reply:
But you are not posting any value from the VM

Reply:
SocialLinksFragment.kt: 83
To show no internet layout.

Reply:
LiveData should only be mutated from VM

Reply:
Oh ok. I should make set fun for it?

Reply:
No, check connection stuff from the VM itself

Reply:
done..